Dockerfile

```diff
@@ -92,6 +93,24 @@ RUN pip3 install -U testresources setuptools
 # Install simple-pid
 RUN pip3 install simple-pid
 
+# Install ROS source dependencies
+RUN mkdir -p /tmp/tmp_ws/src
+COPY carma-base.repos /tmp/tmp_ws/
+RUN vcs import --input /tmp/tmp_ws/carma-base.repos /tmp/tmp_ws/src/
+RUN . /opt/ros/noetic/setup.sh && \
+    cd /tmp/tmp_ws/ && \
+    catkin_make_isolated --install --install-space /opt/ros/noetic --only-pkg-with-deps \
```

Comment:
Why use only-pkg-with-deps here? Since the workspace only contains the contents of the .repos file it seems redundant to list the packages here.

Reply:
The repos might have packages that we don't need and therefore don't have dependencies for. I've tried not specifying specific packages when building some C1T hardware driver packages, but the dependency tree essentially "exploded" with dependencies that did not contribute to the package.

Reply:
Very well, we will work with this for now then.